examples/transformers/MLFlow_X_HuggingFace_Finetune_a_text_classification_model.ipynb

```diff
@@ -0,0 +1,5244 @@
+      "metadata": {
+        "id": "p-aMBdjF5sXf"
+      },
+      "source": [
+        "Create the evaluation metric to log. Loss is also logged, but adding other metrics such as accuracy can make modeling performance easier to understand. For classification task, we use `accuracy` as the tracking metric."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 49,
+          "referenced_widgets": [
+            "71e919cdc65a4d65909e96a0fbd0414c",
+            "eb3ae3fa74ce40418e5fb34210fb1ae6",
+            "8b0a557d91754f72bfd86afec663e7c3",
+            "02f7e8334f0148efa0731c7625ced517",
+            "6509b5ad41b24833bccd73cead1ba8f6",
+            "d775df6bf1554d61ac3ba076a626183d",
+            "10ac297f68ff4a0ba1568d4abc484e6c",
+            "e81a017477b14b4fbc3fb8c092805e51",
+            "aaa7bf8e89654a7caee403352ad7ee28",
+            "33ed6a3b53c64993be80eb8118e40e3d",
+            "a15fcc29f7dc4c9c81a08fe584946c6b"
+          ]
+        },
+        "id": "WyTEqrlC4xnY",
+        "outputId": "2441ecb7-d379-433c-d54e-d945906119ea"
+      },
+      "outputs": [],
+      "source": [
+        "import numpy as np\n",
+        "import evaluate\n",
+        "\n",
+        "metric = evaluate.load(\"accuracy\")\n",
+        "def compute_metrics(eval_pred):\n",
+        "    logits, labels = eval_pred\n",
+        "    predictions = np.argmax(logits, axis=-1)\n",
+        "    return metric.compute(predictions=predictions, references=labels)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "w8BTW-gH6Cg6"
+      },
+      "source": [
+        "Let's set up the trainer! We need to configure two things:\n",
+        "- `TrainingArguments`: it sets hyperparameters like batch size, logging frequency and so. Please refer to [transformers documentation](https://huggingface.co/docs/transformers/main_classes/trainer#transformers.TrainingArguments)\n",
+        "for the full arg list. Don't panick on the long list of args, usually we just need a few out of that.\n",
+        "- `Trainer`: `Trainer` hooks things together, including the model, training args, training/evaluation dataset, and evaluation function."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 8,
+      "metadata": {
+        "id": "QzPUM6KYNj3S"
+      },
+      "outputs": [],
+      "source": [
+        "from transformers import TrainingArguments, Trainer\n",
+        "\n",
+        "# Checkpoints will be output to this `training_output_dir`.\n",
+        "training_output_dir = \"sms_trainer\"\n",
+        "training_args = TrainingArguments(\n",
+        "    output_dir=training_output_dir,\n",
+        "    evaluation_strategy=\"epoch\",\n",
+        "    per_device_train_batch_size=8,\n",
+        "    per_device_eval_batch_size=8,\n",
+        "    logging_steps=8,\n",
```

Comment:
Can we add `num_train_epochs=3` here, so that when we want to run this notebook faster for debugging, we can just reduce `num_train_epochs` to 1 rather than wondering how to reduce the number of epochs.

Reply:
good call! added